pkg/profiler/profiler.go

```diff
@@ -0,0 +1,34 @@
+package profiler
+
+import (
+	"fmt"
+	"time"
+)
+
+type runtime struct {
+	duration   time.Duration
+	sectorName string
+}
+
+type Profiler struct {
+	runtimes []runtime
+}
+
+func NewProfiler() *Profiler {
+	return &Profiler{}
+}
+
+func (p *Profiler) AddRuntime(startTime time.Time, sectorName string) {
+	sectorDuration := time.Since(startTime)
+	p.runtimes = append(p.runtimes,
+		runtime{
+			duration:   sectorDuration,
+			sectorName: sectorName})
+}
+
+func (p *Profiler) PrintRuntimes() {
```

Comment:
There's a tableprinter you can probably use here

Reply:
Going to defer this to until after we get the rest of the CLI profiled